sdk/eventhub/event-hubs/src/eventHubConsumerClient.ts

```diff
@@ -360,7 +366,8 @@ export class EventHubConsumerClient {
    * @throws AbortError if the operation is cancelled via the abortSignal.
    */
   getPartitionIds(options: GetPartitionIdsOptions = {}): Promise<Array<string>> {
-    return this._context
+    return this._contextManager
+      .getGatewayConnectionContext()
```

Comment:
Same comment here - should .managementSession just be a property on _contextManager?